app/controllers/idv/doc_auth_controller.rb

```diff
@@ -50,5 +51,15 @@ def do_meta_refresh(meta_refresh_count)
     def flow_session
       user_session['idv/doc_auth']
     end
+
+    def add_unsafe_eval_to_capture_steps
+      return unless %w[front_image back_image].include?(params[:step])
+
+      # required to run wasm until wasm-eval is available
+      SecureHeaders.append_content_security_policy_directives(
+        request,
+        script_src: ['\'unsafe-eval\''],
```

Comment:
can we verify this in the controller spec?

Reply:
Yes, will do